package.json

```diff
@@ -18,6 +22,8 @@
     "expo-blur": "~11.0.0",
     "expo-status-bar": "~1.2.0",
```

Comment:
Pelo que eu olhei, não está sendo o usado o expo-status-bar no projeto. Acho q pode tirar essa dependência do projeto para ter menos pacotes baixados na hora da instalação

Reply:
Bem observado, já vem com o expo isso, mas não to usando. Removi e não quebrou nada.
[commit](https://github.com/alura-cursos/react-native-criando-testes-para-sua-aplicacao/commit/8682559ccb762ff4ce37178118712c2bbc52487b)